gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -149,6 +150,18 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         .getAddFeatureButtonClicks()
         .as(autoDisposable(this))
         .subscribe(viewModel::onAddFeatureButtonClick);
+    mapContainerViewModel
+        .getAddPolygonPointButtonClicks()
+        .as(autoDisposable(this))
+        .subscribe(mapContainerViewModel::onAddPolygonPointButtonClick);
```

Comment:
Does this refer to the "add point" button on the polygon drawing tools? If so, maybe rename `onAddVertexButtonClick`? would be clearer?

Also, instead of representing the clicks as a stream, can we just call mapContainerViewModel.onAddVertexButtonClick() directly from a binding in the layout?

Reply:
updated and moved to layout